users/firewall.rst

```diff
@@ -3,6 +3,12 @@
 Firewall Setup
 ==============
 
+Local Discovery
+-------------
+If your devices aren't mobile, you can replace the default ``dynamic`` setting for 
+Addresses with something like: ``tcp://192.168.1.xxx:22000, dynamic``
```

Comment:
By keeping the `dynamic`, device can still be mobile. A different requirement for this to work is that devices have/get the same local IP all the time.